src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+
+.search-bar {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  width: 100%;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-search__img {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.small-search_img {
+  height: 11px;
+  width: 11px;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+}
+
+.big-search__input {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small-search__input {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
need to remove outline 😉

<img width="198" alt="image" src="https://user-images.githubusercontent.com/55802630/172438408-7fadfda2-2c43-46a7-9757-d95f547eec87.png">